File: src/frontend/src/app/components/atendimento/tabelaregra/model/tabela-regra-grid-dto.ts
import { Exclude, Expose } from 'class-transformer';

@Exclude()
export class TabelaRegraGridDTO {
  @Expose() id?: string;
  @Expose() convenioNome?: string;
  @Expose() convenioCategoriaNome?: string;
  @Expose() tabelaNome?: string;
  @Expose() createdAt?: string;
  @Expose() deleted?: boolean;
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/atendimento/tabelaregra/dto/TabelaRegraGridDTO.java
package br.com.grupopipa.gestaointegrada.atendimento.tabelaregra.dto;

import java.time.LocalDateTime;
import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.GridDTO;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Builder
@Getter
@Setter
public class TabelaRegraGridDTO implements GridDTO {
    private UUID id;
    private String convenioNome;
    private String convenioCategoriaNome;
    private String tabelaNome;
    private LocalDateTime createdAt;
    private Boolean deleted;
}
